server/punishments.js

```diff
@@ -42,9 +42,11 @@ const AUTOLOCK_POINT_THRESHOLD = 8;
 
 /**
  * @typedef {object} PunishmentEntry
- * @property {string[]} keys
+ * @property {string[]} ips
+ * @property {string[]} alts
```

Comment:
`userids` is probably a better name than `alts` here. It's clearer in terms of _what_ about the alts are being stored, as well as in terms of including the user's original userid as well as alts'.

Reply:
Done here: https://github.com/Zarel/Pokemon-Showdown/pull/5599/commits/4cd73ac99f8aab1eaeb18fc3c4deadbb56bc9718.